ego/cli/sign_test.go

```diff
@@ -276,44 +276,43 @@ type signRunner struct {
 }
 
 func (s signRunner) Run(cmd *exec.Cmd) error {
```

Comment:
```suggestion
func (signRunner) Run(cmd *exec.Cmd) error {
```
For consistency with `Output`.